R_script_HW03.R

```diff
@@ -0,0 +1,77 @@
+library(readxl)
+library(ggplot2)
+my_data <- read_excel("Conv_data_for_HW03.xlsx", sheet = 1)
+
+fCategory <- as.factor(my_data$Category)
+fSex <- as.factor(my_data$Sex)
+Transit_time <- my_data$`transit-time`
+Transit_rate <- my_data$`transit-rate`
+Pellets_per_hour <- my_data$`pellets/hr`
+Pellet_capacity <- my_data$`pellet-capacity`
+Total_intestine_length_cm <- my_data$`total-intestinal-length`
+Small_intestine_length_cm <- my_data$`before-cecum`
+Colon_length_cm <- my_data$`colon-length(cm)`
+Cecum_length_cm <- my_data$`cecum-length(cm)`
+my_data_with_factors <- cbind(my_data, Transit_time, fCategory, fSex)
+
```

Comment:
This code was to rename all of the columns in the Excel sheet? I can see why renaming it might be useful for you to call on those columns in the future.

Reply:
Yep, essentially I made each column an object that I could call in the future. Maybe not necessary if the original column names are sufficiently descriptive. I don't think the actual column names in the dataframe changed though.